### PR TITLE
Update dep anki ankirspy to 2.1.36 and fix the requirments file create

### PR DESCRIPTION
**Pull request by cecini**
fix the requirements file create: 
Add comment header to requirements.txt #41  

update anki-sync-server version to 2.2.0 in pyproject.toml

update dep anki ankirspy to 2.1.36

### CODE REVIEW

**Review by VikashKothary — CHANGES_REQUESTED** [review]
This is great. Just a minor change to the lock script and it'll be good to go.

File: scripts/lock.sh [hunk 1]
```diff
@@ -6,4 +6,9 @@ poetry lock
 poetry export --without-hashes -f requirements.txt > src/requirements.txt
 poetry export --dev --without-hashes -f requirements.txt > src/requirements-dev.txt
 
-echo "-e src/." >> src/requirements-dev.txt
\ No newline at end of file
+echo "-e src/." >> src/requirements-dev.txt
+
+echo -e "THE FILE WAS GENERATED BY POETRY, DO NOT EDIT!\n\n$(cat src/requirements.txt)" > src/requirements.txt
```
[the remark this answers is not in the document]
Comment (cecini) [reply scripts/lock.sh]: Done.

File: pyproject.toml [hunk 2]
```diff
@@ -1,12 +1,12 @@
 [tool.poetry]
 name = "anki-sync-server"
-version = "0.1.0"
+version = "2.2.0"
```
[the remark this answers is not in the document]
Comment (VikashKothary) [reply pyproject.toml:3]: Amazing!

File: scripts/lock.sh [hunk 3]
```diff
@@ -2,8 +2,12 @@
 # file: lock.sh
 # description: Lock dependencies and export requirements.
 
+echo "THE FILE WAS GENERATED BY POETRY, DO NOT EDIT!\n\n" > src/requirements.txt
+echo "THE FILE WAS GENERATED BY POETRY, DO NOT EDIT!\n\n" > src/requirements-dev.txt
```
Comment (upsuper) [inline scripts/lock.sh:6]: Hey. I'm not sure what should I expect, but this doesn't seem to work for me with pip 20.0.2 on Python 3.8. It prints:
```
ERROR: Invalid requirement: 'THE FILE WAS GENERATED BY POETRY, DO NOT EDIT!' (from line 1 of src/requirements.txt)
```

I suppose this line should be commented like
```sh
echo "# THE FILE WAS GENERATED BY POETRY, DO NOT EDIT!\n\n" > src/requirements.txt
echo "# THE FILE WAS GENERATED BY POETRY, DO NOT EDIT!\n\n" > src/requirements-dev.txt
```
instead?

Comment (VikashKothary) [reply scripts/lock.sh:6]: Hi @upsuper,

That's a very good point. Those lines should indeed be comments. 

Could you possibly create a PR for this fix? If not, I'll try to find some time this weekend to fix this.

**Pull request merged**